sql/src/main/java/io/crate/metadata/doc/DocIndexMetaData.java

```diff
@@ -703,4 +705,22 @@ public ColumnPolicy columnPolicy() {
     public Set<Operation> supportedOperations() {
         return supportedOperations;
     }
+
+    private IndexMetaData mergeIndexMetaData(IndexMetaData other) {
+        IndexMetaData.Builder builder = IndexMetaData.builder(other.getIndex().getName())
```

Comment:
can you add a short comment explaining why `builder(IndexMetaData other)` can't be used ?